modules/common/src/main/java/com/ibm/cloud/cloudant/common/package-info.java

```diff
@@ -13,7 +13,7 @@
 
 /**
  * This package contains non-API classes used by
- * [@link com.ibm.cloud.cloudant.v1.Cloudant} to customize requests. User code should
+ * <a href="../v1/Cloudant.html"><code>Cloudant</code></a> to customize requests. User code should
```

Comment:
Tried out the `<detecetLinks>` config and it worked. Thanks! Changed in: 7094a51

Reply:
Update with `mvn verify` still cause problems: https://travis-ci.com/github/IBM/cloudant-java-sdk/jobs/461112349
So I rather revert back to the HTML anchor solution.

Reply:
that's fine, but I'm also OK if you want to just delete the link - I suspect we'll need to anyway when there are other client models that use this common code (because they won't be guaranteed to be `v1`). The important part of this whole statement is that user code shouldn't call the classes.

Reply:
For the document creation in the current state there is no need to change anything on `cloudant-java-sdk` repo. However:
a) I can open an issue for this problem
b) I can open a PR to remove doc generation related scripts and there I can do this little modification. 
c) I can create 2 PR for this 2 purposes

Just this linking problem is on the opening page and it just bothers my eyes 🙂

Reply:
Opened an issue for this problem: https://github.com/IBM/cloudant-java-sdk/issues/65